core/sql/cli/Cli.cpp

```diff
@@ -9717,7 +9717,7 @@ Lng32 SQLCLI_LOBcliInterface
 	// set parserflags to allow ghost table
 	currContext.setSqlParserFlags(0x1);
 	
-	Int64 numChunks = 0;
+	Lng32 numChunks = 0;
```

Comment:
This can cause corruption. The len coming back will be 8. So there is a potential for corruption. The rowcount which is the container needed to hold the return value needs to be large enough to hold the return value always.